src/Tools/dotnet-format/dotnet-format.csproj

```diff
@@ -6,6 +6,7 @@
     <RootNamespace>Microsoft.CodeAnalysis.Tools.CodeFormatter</RootNamespace>
     <Description>Command line tool for formatting code files.</Description>
 
+    <ServerGarbageCollection>true</ServerGarbageCollection>
```

Comment:
interesting.  how come?

Reply:
Workstation GC is better for single core machines and in some cases 32-bit processes (neither applies), and in some cases for single-threaded algorithms (no longer applies). If you are seeing GC overhead in profiling scenarios, it's probably because you used the slow one.

Reply:
Cool.  Thanks!